packages/go_router_builder/tool/run_tests.dart

```diff
@@ -59,3 +63,20 @@ Future<void> main() async {
     }, timeout: const Timeout(Duration(seconds: 100)));
   }
 }
+
+Future<Version> _packageVersion() async {
+  final PackageConfig packageConfig =
+      await loadPackageConfigUri(Isolate.packageConfigSync!);
+
+  final String pkgPath = Platform.script.resolve('../').path;
```

Comment:
I suspect the `/` here is what's breaking the test on Windows.

Reply:
Good try. Let me hack.